src/theme/defaultTheme/tag.ts

```diff
@@ -1,11 +1,22 @@
 export const tag = {
   base:
-    "lib:font-sans lib:font-semibold lib:bg-gray-100 lib:text-gray-800 lib:inline-flex lib:items-center lib:justify-center lib:appearance-none lib:rounded-md lib:transition-all lib:relative lib:whitespace-nowrap lib:align-middle lib:outline-none lib:w-auto lib:select-none lib:disabled:cursor-not-allowed lib:disabled:opacity-40",
-  prefix: "lib:flex lib:mr-2",
-  suffix: "lib:flex lib:ml-2",
+    "lib:bg-gray-100 lib:text-gray-800 lib:inline-flex lib:items-center lib:justify-center lib:appearance-none lib:transition-all lib:whitespace-nowrap lib:align-middle lib:outline-none lib:select-none",
   size: {
-    xs: "lib:h-6 lib:min-w-6 lib:text-xs lib:px-2",
-    sm: "lib:h-8 lib:min-w-8 lib:text-sm lib:px-3",
-    lg: "lib:h-10 lib:min-w-10 lib:text-base lib:px-4 lib:rounded-lg",
+    xs:
+      "lib:h-6 lib:w-auto lib:min-w-6 lib:px-2 lib:text-xs lib:rounded-md lib:font-medium",
+    sm:
+      "lib:h-8 lib:w-auto lib:min-w-8 lib:px-3 lib:text-sm lib:rounded-md lib:font-medium",
+    lg:
+      "lib:h-10 lib:w-auto lib:min-w-10 lib:px-4 lib:text-sm lib:rounded-lg lib:font-medium",
```

Comment:
Moved all the size related classes from base to here.
We can remove the repeated one's after we the states design(disabled, active etc.,)